commits.go

```diff
@@ -35,23 +35,24 @@ type CommitsService struct {
 //
 // GitLab API docs: https://docs.gitlab.com/ce/api/commits.html
 type Commit struct {
-	ID             string           `json:"id"`
-	ShortID        string           `json:"short_id"`
-	Title          string           `json:"title"`
-	AuthorName     string           `json:"author_name"`
-	AuthorEmail    string           `json:"author_email"`
-	AuthoredDate   *time.Time       `json:"authored_date"`
-	CommitterName  string           `json:"committer_name"`
-	CommitterEmail string           `json:"committer_email"`
-	CommittedDate  *time.Time       `json:"committed_date"`
-	CreatedAt      *time.Time       `json:"created_at"`
-	Message        string           `json:"message"`
-	ParentIDs      []string         `json:"parent_ids"`
-	Stats          *CommitStats     `json:"stats"`
-	Status         *BuildStateValue `json:"status"`
-	LastPipeline   *PipelineInfo    `json:"last_pipeline"`
-	ProjectID      int              `json:"project_id"`
-	WebURL         string           `json:"web_url"`
+	ID             string                  `json:"id"`
+	ShortID        string                  `json:"short_id"`
+	Title          string                  `json:"title"`
+	AuthorName     string                  `json:"author_name"`
+	AuthorEmail    string                  `json:"author_email"`
+	AuthoredDate   *time.Time              `json:"authored_date"`
+	CommitterName  string                  `json:"committer_name"`
+	CommitterEmail string                  `json:"committer_email"`
+	CommittedDate  *time.Time              `json:"committed_date"`
+	CreatedAt      *time.Time              `json:"created_at"`
+	Message        string                  `json:"message"`
+	ParentIDs      []string                `json:"parent_ids"`
+	Stats          *gitlab.CommitStats     `json:"stats"`
```

Comment:
I see you added `gitlab` in a few places. That will not work as this _is_ the `gitlab` package`. So please revert that...